### DISCUSSION TITLE
Help with plugins

### DISCUSSION THREAD

**Discussion by weirlive** [squidfunk/mkdocs-material#5158] [General]
I'm very new to Mkdocs and the Materials Insiders section. I paid today and got accepted to the repo. I followed the install instructions which seem to have worked properly. Now When I try to use any of the plugins I get the following:

-  Config value 'plugins': The "blog" plugin is not installed

this is the same for any plugin. How do I install these?

**Comment by weirlive** [comment]
And of course I found it.. 

https://www.mkdocs.org/dev-guide/plugins/

**Reply by squidfunk** [reply]
Thanks for sponsoring! You should install Material for MkDocs Insiders [as mentioned in our documentation](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-pip):

```
pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
```

This will also make sure that all built-in plugins are properly installed.

**Reply by weirlive** [reply]
interesting thats how I did it but these plugins don't seem to be installed.

**Reply by weirlive** [reply]
okay so... ran it again and now its all working... oh Monday.. :D 

Thank you for making such an amazing tool!!

**Reply by squidfunk** [reply]
Glad you solved it!
